primitives/account/src/staking_contract/staker.rs

```diff
@@ -203,7 +215,7 @@ impl StakingContract {
         db_txn: &mut WriteTransaction,
         staker_address: &Address,
         value: Coin,
-    ) -> Result<(), AccountError> {
+    ) -> Result<Vec<Log>, AccountError> {
```

Comment:
I think it would be more consistent to return 
```suggestion
    ) -> Result<AccountInfo, AccountError> {
```
as almost all other functions in this file do with the respective `*Info` type